_field-types/mapping-parameters/eager_global_ordinals.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Eager global ordinals
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 35
+has_children: false
+has_toc: false
+---
+
+# Eager global ordinals
+
+The `eager_global_ordinals` mapping parameter controls when global ordinals are built for a field. When enabled, global ordinals are computed during index refresh rather than "lazily" during query execution. This can improve performance for operations that rely on global ordinals, for example sorting and aggregations on keyword fields, however it may also increase index refresh times and memory usage.
+
+Global ordinals represent a mapping from term values to integer identifiers and are used internally to quickly execute aggregations and sort operations. By loading them "eagerly", the system reduces query latency at the cost of additional upfront processing during indexing.
+
+By default `eager_global_ordinals` are disabled, ensuring the cluster is optimized for indexing speed.
+
+Global ordinals are stored in the field data cache and consume heap memory. Fields with high cardinality can consume a large amount of heap memory. To prevent memory-related issues, it is important to carefully configure the [Field data circuit breaker settings]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/circuit-breaker/#field-data-circuit-breaker-settings)
```

Comment:
```suggestion
Global ordinals are stored in the field data cache and consume heap memory. Fields with high cardinality can consume a large amount of heap memory. To prevent memory-related issues, it is important to carefully configure the [field data circuit breaker settings]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/circuit-breaker/#field-data-circuit-breaker-settings).
```